QKSMS/src/main/res/values-pl/strings.xml

```diff
@@ -359,44 +354,41 @@
-  <string name="pref_category_sending_mms">Sending MMS</string>
-  <string name="pref_category_mms_configuration">MMS Configuration</string>
-  <string name="pref_mms_contact_support_summary">Contact the QKSMS team for help setting up MMS</string>
-  <string name="pref_markdown_enabled">Text formatting</string>
-  <string name="pref_markdown_enabled_summary">Use markdown syntax to get bold, underline, and italics in your messages</string>
-  <string name="want_read_indicators">Want read indicators?</string>
+  <string name="sending_blocked_welcome_screen_message">Bez pośpiechu! Już za chwilę będziesz mógł wysyłać SMS-y!</string>
+  <string name="notification_colored_icon_ready">Naciśnij, aby dodać skrót do ekranu głównego.</string>
+  <string name="notification_title_colored_icon_ready">Kolorowa ikona QKSMS jest już gotowa</string>
+  <string name="icon_ready">Twoja nowa ikona jest gotowa!</string>
+  <string name="icon_ready_message">Możesz ją teraz ponownie dodać do ekranu głównego z nowym kolorem.</string>
+  <string name="dont_ask_again">Nie pytaj ponownie</string>
+  <string name="mms_setup_set_up_mms_title">Skonfigurować MMS?</string>
+  <string name="mms_setup_set_up_mms_body">Wygląda na to, że nie masz skonfigurowanych MMS w QKSMS. Naciśnij OK, aby automatycznie skonfigurować MMS.</string>
+  <string name="mms_setup_setting_up_later_title">Skonfiguruj ustawienia MMS później.</string>
+  <string name="mms_setup_setting_up_later_body">Przejdź do sekcji MMS w Ustawieniach programy, aby spróbować ponownie automatycznej konfiguracji, skontaktuj się ze wsparciem lub wprowadź własne ustawienia.</string>
+  <string name="mms_setup_no_configurations_found_title">Nie znaleziono konfiguracji</string>
+  <string name="mms_setup_no_configurations_found_body">Nie byliśmy w stanie znaleźć konfiguracji MMS dla Twojego operatora. Naciśnij OK, aby skontaktować się ze wsparciem. Pomożemy Ci to naprawić!</string>
+  <string name="mms_setup_one_configuration_found_title">Znaleziono konfigurację</string>
+  <string name="mms_setup_one_configuration_found_body">Znaleziono konfigurację dla operatora %s. Wygląda w porządku?</string>
+  <string name="mms_setup_multiple_configurations_found_title">Znaleziono wiele konfiguracji</string>
+  <string name="mms_setup_multiple_configurations_found_body">Znaleziono kilka konfiguracji MMS. Wybierz swojego operatora lub naciśnij Brak jeśli nie znajdziesz go na liście.</string>
+  <string name="mms_setup_next_steps_title">Następne kroki</string>
+  <string name="mms_setup_next_steps_body">Przejdź do sekcji MMS w Ustawieniach programy, jeśli chcesz spróbować ponownie automatycznej konfiguracji, aby skontaktuj się ze wsparciem lub by wprowadzić własne ustawienia.</string>
+  <string name="mms_setup_success_body">Przejdź do sekcji MMS w Ustawieniach programy, aby spróbować ponownie automatycznej konfiguracji, skontaktuj się ze wsparciem lub wprowadź własne ustawienia.</string>
```

Comment:
To reflect the English version fully, I'd change this to:
'Jeśli napotkasz błędy, przejdź do sekcji MMS w ustawieniach programu, aby ponownie spróbować automatycznej konfiguracji, skontaktować się ze wsparciem lub wprowadzić własne ustawienia.'